web/documentserver-example/ruby/app/controllers/home_controller.rb

```diff
@@ -395,6 +395,25 @@ def reference
         end
       end
 
+      if fileName.empty? and body.key?("link")
+        if !body["link"].include?(DocumentHelper.get_server_url(false))
+          data = {
+            :url => body["link"],
```

Comment:
Prefer to use the new syntax `new(a: 1)` instead of `new(:a => 1)`.

Reply:
> Prefer to use the new syntax `new(a: 1)` instead of `new(:a => 1)`.

I used the syntax that the rest of the code is written in. I can use a new hash syntax if needed, but I didn't find the use of "new(a: 1)" in the Ruby example.

Reply:
I get your point. But I still think it's better to use the new syntax because when it comes time to refactor the codebase, some of that's written will already be in the new format.

Reply:
> I get your point. But I still think it's better to use the new syntax because when it comes time to refactor the codebase, some of that's written will already be in the new format.

I agree. Rewritten using the actual syntax.